### PR TITLE
Better topology examples

### PR DESCRIPTION
**Pull request by EnessenE**
This closes nothing.

**Note to reviewers:** remember to look at the commits in this PR and consider if they can be squashed
**Note to contributors:** remember to re-generate client set if there are any API changes

## Summary Of Changes
Simply add some additional comments and examples to the topology examples made available.

### CODE REVIEW

File: docs/examples/exchanges/fanout-exchange-with-args.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: fanout-alt-exchange # name of the object in kubernetes
+spec:
+  name: fanout-alt-exchange # name of the exchange
+  vhost: "/test-vhost" # default to '/' if not provided
+  type: fanout # default to 'direct' if not provided
+  autoDelete: false
+  durable: true
+  arguments: # additional to provide towards the exchange, may be left empty.
+    - alternate-exchange: direct-exchange
```
Comment (Zerpet) [inline docs/examples/exchanges/fanout-exchange-with-args.yaml:14]: This is an object, not an array.

```suggestion
  arguments: # additional to provide towards the exchange, may be left empty.
    alternate-exchange: direct-exchange
```

**Review by Zerpet — CHANGES_REQUESTED** [review]

File: docs/examples/exchanges/fanout-exchange-with-args.yaml (hunk 1 above)
Comment (EnessenE) [reply docs/examples/exchanges/fanout-exchange-with-args.yaml:14]: You are completely right! I didn't get around to fixing this yet as I wanted to go over the examples one more time. But accepted the change

**Review by Zerpet — APPROVED** [review]

**Comment by Zerpet** [comment]
We will have to wait for the CLA approval before merging.  Thank you for this contribution!

**Comment by EnessenE** [comment]
Thanks for the quick review!

**Pull request merged**